libr/core/cmd_print.c

```diff
@@ -4324,7 +4324,7 @@ static void cmd_print_bars(RCore *core, const char *input) {
 						k++;
 					}
 					break;
-				case 'f':
+				case 'F':
```

Comment:
i think its better to keep p=f instead of p=F, uppercase chars shouldnt be used unless there's no other lowercase letter left. the only uppercase one is p=A and i would like to also keep it lowercase, so im open to suggestions. can you fix the help message instead ?

Reply:
pzf does count of flags, pzF does of 0xFF so why not mimic it too with prc=f for flags and prc=F for 0xFF ?
the prc=F is mentioned even in book iiuc

Reply:
oh, ok then it makes sense, but we should have p=f to handle the flags case to be in sync and maybe mention the pz[*] in the help to clarify this too. merging!